Jbarget/oxymore
# replaces placeholder text and polishes preview page

### What changes have you made?
- added in the translated copy for each of the Conscious Shopping interviews
- made some tweaks on the project preview page to remove a weird scroll on desktop
- the carousel can now be moved using the arrow 

### Which issue(s) does this PR fix?
Fixes #287 
Fixes #208 
Fixes #265 

### Screenshots (if there are design changes)
<img width="297" alt="Screenshot 2021-01-01 at 18 22 43" src="https://user-images.githubusercontent.com/53219789/103444224-9677c300-4c5e-11eb-8f62-7762f59e0b1a.png">


### How to test
- Move the date back in the Conscious Shopping page to be able to see the project preview page
- Check that the translation works on each of the different interviews and the arrow changes the interview on click
- Not picked up any issues on Safari, double-check on your side to make sure there are no sneaky display bugs
